test/pythis_tutorial.py

```diff
@@ -0,0 +1,295 @@
+'''Example configuration file for an ee->ZH->mumubb analysis in heppy, with the FCC-ee
```

Comment:
Now starting to comment! Thanks for the new PR.
I'll be a bit terse, thanks for your understanding :-)

typo in the name of this file pythis -> pythia